cli/state/forms.py

```diff
@@ -38,4 +38,5 @@ class GetStateListForm(Schema):
             validate.Range(min=1, error='Limit must be greater than 0.'),
         ],
     )
-    node_url = NodeUrlField(required=False)
+    reverse = fields.Boolean(required=False)
```

Comment:
What a sense of it. Give, please, example when it cant be a bool.

Reply:
@Alladin9393, when you create `bool` argument using click — https://github.com/Remmeauth/remme-core-cli/blob/6a65cec8455b65fb770e6cd8f69a4db75c493932/cli/state/cli.py#L74, you will get `bool` type in the according to the argument — https://github.com/Remmeauth/remme-core-cli/blob/6a65cec8455b65fb770e6cd8f69a4db75c493932/cli/state/cli.py#L77.